examples/networkattachmentdefinition.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
please rename this file to ovs-net-vlan100.yaml and also the name to "ovs-net-vlan100".
there could be multiple network attachment definitions, even of the same plugin.

Reply:
Done